performance/class-mime-types-caching.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * MIME Types Caching.
+ *
+ * @package vip-performance
+ */
+
+// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+
+namespace Automattic\VIP\Performance;
+
+/**
+ * MIME Types Caching class.
+ */
+class Mime_Types_Caching {
+	public const MINIMUM_WORDPRESS_VERSION  = '6.4';
+	public const CACHE_GROUP                = 'mime_types';
+	public const MIME_TYPES_CACHE_KEY       = 'vip_mime_types';
+	public const MAX_POSTS_TO_QUERY_DEFAULT = 500000;
+
+	/**
+	 * Class initialization.
+	 */
+	public static function init() {
+		global $wp_version;
+
+		if ( isset( $wp_version ) &&
+			version_compare( $wp_version, self::MINIMUM_WORDPRESS_VERSION, '>=' ) ) {
```

Comment:
https://developer.wordpress.org/reference/functions/is_wp_version_compatible/ would have been slightly better to use here, avoiding the need to pull in the global variable and account for version suffixes.